kthfsdv-plotting/main.py

```diff
@@ -0,0 +1,128 @@
+import numpy as np
+import matplotlib.pyplot as plt
+from matplotlib.animation import FuncAnimation
+import matplotlib
+from matplotlib import pyplot as plt
+from matplotlib.widgets import Button, TextBox
+import pandas as pd 
+
+class Exc2():
```

Comment:
Try to always leave a blank line between the class definition and the first method. Moreover, you can avoid parentheses if your class doesn't extend anything.
```suggestion
class Exc2:
```